app/build.gradle.kts

```diff
@@ -96,6 +96,9 @@ dependencies {
     implementation("androidx.navigation:navigation-compose:2.7.5")
     implementation("androidx.lifecycle:lifecycle-runtime-compose:2.6.2")
 
+    // Accompanist
+    implementation("com.google.accompanist:accompanist-permissions:0.32.0")
```

Comment:
I got careful with using Accompanist libraries. They can help, but a lot of them got deprecated already since their mechanism gets built into the normal Compose API. Nothing to change here, just for future reference in case you're working on any real projects